python-sdk/tests_integration/astro_deploy/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM quay.io/astronomer/astro-runtime:7.1.0-base
```

Comment:
@pankajastro would it make sense to use an earlier version of the Runtime (>= 7.2.0), which has the Python SDK included?

Reply:
yup, we should use the latest runtime image

Reply:
Updated to 7.2.0